web/docs/tutorials/dev-excuses-app/creating-the-project.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Creating the project
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+By now you're already know, [how to create a project and do have Wasp installed](/getting-started.md). So let’s create a new web app project named `ItWaspsOnMyMachine`.
+
+```
+wasp new ItWaspsOnMyMachine
+```
+
+Changing the working directory:
+```
+cd ItWaspsOnMyMachine
+```
+
+Starting the app:
+```
+wasp start
+```
+
+Now your default browser should open up with a simple predefined text message. That’s it! 🥳 For now – the codebase consists of only two files! `main.wasp` is the config file that defines the application’s functionality. And `MainPage.js` is the front-end. Everything else, like `Main.css` can be ignored.
+
+<img alt="Initial page"
```

Comment:
This image doesn't look the best - could we reduce the width of the left sidebar, I think it doesn't need so much space? That might also make the code better visible.